packages/security/src/components/__tests__/__snapshots__/public-api-test.js.snap

```diff
@@ -3555,6 +3571,7 @@ Map {
             "danger--ghost",
             "danger--tertiary",
             "tertiary",
+            "ghost-danger",
```

Comment:
Not quite sure if my change is relevant to the prop type of `FileUploader`(and several snapshot mismatches below). It seems `ghost-danger` is not part of carbon button kind but manually inserted at https://github.com/liunate/ibm-cloud-cognitive/blob/a04bc55da2767e10605fcd7e79bd617dc9a6c4af/packages/security/src/components/Button/Button.js#L20
https://github.com/carbon-design-system/ibm-cloud-cognitive/blob/a04bc55da2767e10605fcd7e79bd617dc9a6c4af/packages/security/src/components/Button/Button.js#L20

In that case, could these `buttonKind` snapshot mismatches be something not updated back at when the line I just mentioned 👆was added?

Reply:
> Hello 👋 while I am migrating the new decorator code base, I am wondering whether we need some feature flag for people don't want the new round-shaped decorator?

That said, there is a property `<Decorator noBorderRadius>` which renders the original square-shaped decorator, it's just not `true` by default. 🤷

Reply:
Or just the opposite, should I just make it square-shaped by default, and expose some property like `<Decorator pillShaped>` so ppl want to keep original decorator look don't need to make any change at `@carbon/ibm-security` upgrade?